java/core/src/test/java/co/worklytics/psoxy/gateway/impl/HealthCheckRequestHandlerTest.java

```diff
@@ -7,6 +7,7 @@
 import co.worklytics.psoxy.gateway.ProxyConfigProperty;
 import co.worklytics.test.MockModules;
 import dagger.Component;
+import org.apache.hc.core5.http.HttpStatus;
```

Comment:
you meant `org.apache.http.HttpStatus;` ??

Reply:
i think the other is newer